### PR TITLE
testimonial ui improved

### PR DESCRIPTION
**Pull request by shonali2600**
## Related Issue
Improved Testimonial UI

Closes: #318 

### Describe the changes you've made
I've added the colour, shadow, border radius, to the card. And added border, shadow and improved hover effect on arrow buttons.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots
codetrophs

### CODE REVIEW

**Comment by shonali2600** [comment]
@aryasoni98 kindly review my PR

**Comment by aryasoni98** [comment]
@shonali2600 Change the Red Colour to White background

**Review by aryasoni98 — COMMENTED** [review]
Change the Red To White Background

File: src/scss/carousel.module.scss [hunk 1]
```diff
@@ -55,6 +55,9 @@
   // margin-top: -7.5px;
   // padding: 15px 15px 40px 15px;
   // background-color: var(--theme-background-darker);
+  background-color: #a0021d;
+  box-shadow: 10px 10px 7px 3px #2c030a;
```
Comment (msk4862) [inline src/scss/carousel.module.scss:59]: change box shadow to `1px 1px 3px #5a5a5a` to maintain uniformity on the page.

**Comment by shonali2600** [comment]
fcard

Now please review, and let me know if any further changes to be made.

**Review by msk4862 — CHANGES_REQUESTED** [review]
@shonali2600 maybe you can convert images to rounded ones and add styled commas at the beginning of blockquote, something similar to [this](https://www.atompilot.com/wp-content/uploads/2017/08/peer-testimonial.jpg)

**Comment by shonali2600** [comment]
@msk4862 I made images rounded, but not able to add quote icon there, can you please help me

**Comment by shonali2600** [comment]
@msk added the quote icon and made the images round, now please review.

**Comment by shonali2600** [comment]
quote

**Comment by msk4862** [comment]
@shonali2600 looks nice, but make sure all checks passes.

**Comment by pinkman7009** [comment]
make sure all the checks pass, and try to add some padding to the box because the picture looks too cramped up.

**Review by pinkman7009 — CHANGES_REQUESTED** [review]
try to add some padding to the box to create some space between the pictures and the border as it looks too cramped up now. and also make sure all the checks pass.

**Comment by shonali2600** [comment]
@pinkman7009 @msk4862 Added the padding, and all checks passed, now please review

**Review by pinkman7009 — APPROVED** [review]
looks good to me :D

**Review by msk4862 — APPROVED** [review]
Looks good :+1:

**Review by Anurag5086 — APPROVED** [review]
Looks good!

**Review by aryasoni98 — APPROVED** [review]
Nice 👍

**Pull request merged**